src/main/pages/user-management/permissions.adoc

```diff
@@ -50,135 +48,141 @@ The following permissions are applicable to organizations:
+|update |Allows updating of the stack configuration.
 |delete |Allows deleting of the stack.
 |setPermissions |Allows managing permissions for the stack.
 |===
 
 [id="permissions-api"]
 == Permissions API
 
-All permissions can be managed by using the provided REST API. The APIs are documented using Swagger at `[{host}/swagger/#!/permissions]`.
+All permissions can be managed using the provided REST API. The APIs are documented using Swagger at `[{host}/swagger/#!/permissions]`.
 
 [id="list-permissions"]
-== List Permissions
+== Listing permissions
 
-List the permissions which can be applied to a specific resources: `GET /permissions`
+To list the permissions that apply to a specific resources, run this command:
+ `$ GET /permissions`
 
-Applicable `domain` values are the following:
+The `domain` values are:
 
 [cols="",options="header",]
 |===
 |Domain
-|system
-|organization
-|workspace
-|stack
+|`system`
+|`organization`
+|`workspace`
+|`stack`
 |===
 
-Note: `domain` is optional, in this case the API will return all possible permissions for all domains.
+[NOTE]
+====
+`domain` is optional.  In this case, the API returns all possible permissions for all domains.
+====
 
 [id="list-permissions-for-specific-user"]
-== List Permissions for Specific User
+== Listing permissions for a user
 
-List the permissions which are applied to a specific user: `GET /permissions/{domain}`
+To list the permissions that apply to a user, run this command:
+`$ GET /permissions/{domain}`
 
-Applicable `domain` values are the following:
+The `domain` values are:
 
 [cols="",options="header",]
 |===
 |Domain
-|system
-|organization
-|workspace
-|stack
+|`system`
+|`organization`
+|`workspace`
+|`stack`
 |===
 
-`instance` parameter corresponds to the ID of the resource you want to get the applied permissions.
 
 [id="list-permissions-for-all-users"]
-== List Permissions for All Users
+== Listing permissions for all users
 
-List the permissions which are applied to a specific user (you must have sufficient permissions to allow you to see this information): `GET /permissions/{domain}/all`
+[NOTE]
+====
+You must have sufficient permission to see this information.
+====
 
-Applicable `domain` values are the following:
+To list the permissions that apply to all users, run this command:
 
-[cols="",options="header",]
+`GET /permissions/{domain}/all`
+
+The `domain` values are:
+
+[cols="", options="header",]
 |===
 |Domain
-|system
-|organization
-|workspace
-|stack
+|`system`
+|`organization`
+|`workspace`
+|`stack`
 |===
 
-`instance` parameter corresponds to the ID of the resource you want to get the applied permissions for all users.
 
 [id="assign-permissions"]
-== Assign Permissions
+== Assigning permissions
 
-Assign permissions to a resource: `POST /permissions`
+To assign permissions to a resource, run this command:
 
-Applicable `domain` values are the following:
+`POST /permissions`
+
+The `domain` values are:
```

Comment:
Likewise.

The primary reason I mention this is because of the potential for maintenance later on, in which case some things may be missed. If these domain values are only documented in one location, they only need to be _updated_ in one location. :)

Reply:
What about moving   -> The `domain` values  <-  to the Listing permissions heading